callback_plugins/custom_junit.py

```diff
@@ -14,12 +40,15 @@ class CallbackModule(JunitCallbackModule):
     CALLBACK_NAME = 'custom_junit'
 
     def __init__(self):
+        self._defs = None
         super(CallbackModule, self).__init__()
+        self.set_options()
 
-        # Custom environment variable handling
         # Update this to parse these values from the config file, as well as the env.
         self._output_dir = os.path.expanduser("~/.ansible.log")
```

Comment:
This should also be configurable, so we could point it at the expect logs dir in zuul jobs.